lib/tasks/link_governments.rake

```diff
@@ -0,0 +1,13 @@
+namespace :governments do
```

Comment:
It might be good to have this task in the `publishing_api` namespace as we have a few tasks in there already which interact with the Publishing API.